scripts/jest/root.config.js

```diff
@@ -30,4 +30,8 @@ module.exports = {
             lines: 90,
         },
     },
+    // Jest's default reporters are [clover, json, lcov, text]. We add the second text reporter to
+    // use the output as the step summary when running tests in GitHub Actions. (Ideally, we'd use
+    // a markdown reporter, but there don't seem to be any...)
+    coverageReporters: ['clover', 'json', 'lcov', 'text', ['text', { file: 'coverage.txt' }]],
```

Comment:
I took a look at using the [html reporter](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports/lib) instead, but it only seems to give a high-level directory overview rather than per-file (unless you want to deeply traverse into HTML files). So yeah, `text` seems most convenient here.